mui/notification.py

```diff
@@ -21,6 +25,14 @@ def __init__(self):
     def __del__(self):
         UIContext.unregisterNotification(self)
 
+    def OnContextClose(self, context: UIContext) -> None:
+        server_pid: str = subprocess.run(
+            ["lsof", "-t", "-i:50010"], text=True, capture_output=True
+        ).stdout.strip()
+
+        if server_pid:
+            os.kill(int(server_pid), 9)
```

Comment:
This looks a bit hacky. Is there no way to keep track of the actual process handle and pass it to this point?

Reply:
for now I've kept it as an attribute of UINotification, not sure if there's a better way https://github.com/trailofbits/ManticoreUI/blob/360838c1d5e8d3736c93b9768eeecf478c26abda/mui/notification.py#L26